### ISSUE TITLE
mapPlot() needs to silence rgdal messages

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#968]
The screen shot (from the vignette) shows how these messages will annoy users.
![screen shot 2016-05-11 at 12 10 53 pm](https://cloud.githubusercontent.com/assets/99469/15185979/95f4a94c-1771-11e6-9983-e4f504bd559e.png)

**Comment by dankelley** [comment]
Hm. A try block with `silent=TRUE` (as below) doesn't do it. I'll try using `rawTransform` instead.
```R
try({XY <- unname(rgdal::project(cbind(x, y), proj=as.character(.Projection()$projection), inv=TRUE))}, silent=TRUE)

```

**Comment by dankelley** [comment]
A better test (independent of oce)
```R
> rgdal::project(cbind(0,-90), proj="+proj=lcc +lat_1=30 +lat_2=50 +lon_0=-40")
projected point not finite
     [,1] [,2]
[1,]  Inf  Inf
```

**Comment by dankelley** [comment]
Unfortunately, `try(..., silent=TRUE)` does not help
```R
> try(rgdal::project(cbind(0,-90), proj="+proj=lcc +lat_1=30 +lat_2=50 +lon_0=-40"),silent=TRUE)
projected point not finite
     [,1] [,2]
[1,]  Inf  Inf
```

**Comment by dankelley** [comment]
Neither does setting the warnings. The problem is that the C code is printing this directly, so it cannot be intercepted by the R scheme that controls messages
```R
> options(warn=-1);try(rgdal::project(cbind(0,-90), proj="+proj=lcc +lat_1=30 +lat_2=50 +lon_0=-40"),silent=TRUE)
projected point not finite
     [,1] [,2]
[1,]  Inf  Inf
```

**Comment by dankelley** [comment]
We *can* make it quiet with `sink`.
```R
> sink("/dev/null")
> xy <- rgdal::project(cbind(0,-90), proj="+proj=lcc +lat_1=30 +lat_2=50 +lon_0=-40")
> sink()
> xy
     [,1] [,2]
[1,]  Inf  Inf
> 
```

**Comment by dankelley** [comment]
Well, we can do as follows.
```R
> msg<-capture.output(xy <- rgdal::project(cbind(0,-90), proj="+proj=lcc +lat_1=30 +lat_2=50 +lon_0=-40"))
> msg
[1] "projected point not finite"
```

**Comment by dankelley** [comment]
Fixed in `develop` commit 13c103c2e280756a048ca38982303311ace3341f
